### PR TITLE
Code Refactor to Adhere to PEP-8 Function and Variable Naming Conventions

### PR DESCRIPTION
**Pull request by jgarciabu**
I went through all the code and refactored the function and variable names to adhere to PEP-8 naming conventions. CamelCase names have been replaced by lowercase_names_separated_by_underscores for functions and variables. Classes using camelCase have been left untouched as classes can use camelCase per PEP-8.

### CODE REVIEW

File: twweet_cli/Twweeter.py [hunk 1]
```diff
@@ -130,22 +130,22 @@ def get_trending_topics(self):
     def process_or_store(self, tweet):
         print(json.dumps(tweet, indent=1))
 
-    def readTimeLine(self):
+    def read_time_line(self):
```
Comment (omeshomes) [inline twweet_cli/Twweeter.py]: timeline is one word, i think read_timeline would be better

**Review by omeshomes — APPROVED** [review]
This edit looks good! Makes the code much more readable and reusable

**Review by omeshomes — CHANGES_REQUESTED** [review]
Looks like there are conflicts with the base branch--maybe try pulling any changes from remote and commit those changes?

**Comment by jgarciabu** [comment]
Let me know if I need to revisit, but I think it's all set! This is one of my first open source contributions so I appreciate your encouraging words.

**Comment by 0x006E** [comment]
This ought to work! Good work

**Pull request merged**